conf/documentation.conf

```diff
@@ -1130,6 +1130,18 @@ description=<<EOT
 List of LDAP attributes that can be used in the sources configuration
 EOT
 
+[advanced.unreg_onstop_timing]
+type=numeric
+description=<<EOT
+Define the wait timing before unregister a node when receiving an accounting stop request.
+EOT
+
+[advanced.unreg_onstop_ssid]
```

Comment:
That's not dynamic at all...
We need to be able to define more specific rules other than the SSID or at least multiple SSIDs or no filtering at all.

I suggest you create a new scope in the VLAN filters and use it for that.